README.md

```diff
@@ -1,11 +1,133 @@
-<h1 align="center">firestore-meilisearch ⚠ WIP ⚠️</h1>
+<h1 align="center">⚠ WIP ⚠️</h1>
 
 <p align="center">
+  <img src="https://raw.githubusercontent.com/meilisearch/integration-guides/main/assets/logos/meilisearch_firebase.svg" alt="Meilisearch-Firestore" width="200" height="200" />
+</p>
+
+<h1 align="center">Meilisearch Firestore</h1>
+
+<h4 align="center">
+  <a href="https://github.com/meilisearch/meilisearch">Meilisearch</a> |
+  <a href="https://docs.meilisearch.com">Documentation</a> |
+  <a href="https://slack.meilisearch.com">Slack</a> |
+  <a href="https://roadmap.meilisearch.com/tabs/1-under-consideration">Roadmap</a> |
+  <a href="https://www.meilisearch.com">Website</a> |
+  <a href="https://docs.meilisearch.com/faq">FAQ</a>
+</h4>
+
+<p align="center">
+  <a href="https://packagist.org/packages/meilisearch/firestore-meilisearch"><img src="https://img.shields.io/packagist/v/meilisearch/firestore-meilisearch" alt="Latest Stable Version"></a>
+  <a href="https://github.com/meilisearch/firestore-meilisearch/actions"><img src="https://github.com/meilisearch/firestore-meilisearch/workflows/Tests/badge.svg" alt="Test"></a>
+  <a href="https://github.com/meilisearch/firestore-meilisearch/blob/main/LICENSE"><img src="https://img.shields.io/badge/license-MIT-informational" alt="License"></a>
   <a href="https://ms-bors.herokuapp.com/repositories/8"><img src="https://bors.tech/images/badge_small.svg" alt="Bors enabled"></a>
 </p>
 
-## Fulltext Search on Firebase with Meilisearch
+<p align="center">⚡ The Meilisearch API extension written for Firebase</p>
+
+# Search in your Firestore content with Meilisearch
+
+**Author**: Meilisearch (**[https://meilisearch.com](https://meilisearch.com)**)
+
+**Description**: Fulltext Search on Firebase with Meilisearch
+
+
+
+**Details**: Use this extension to synchronize documents from a Cloud Firestore collection to a Meilisearch index. This allows you to use a full text search in your Cloud Firestore documents.
+
+This extension listens to each creation, update or deletion of your documents to keep them sync with your Meilisearch index, so the data in Meilisearch is a mirror of your content in Cloud Firestore. You can then run queries on this mirrored dataset.
+
+Note that this extension only listens for changes to _documents_ in a specific collection, but not changes in any _subcollection_. However, you can install additional instances of this extension to listen to other collections in your Firestore database.
+
+#### Additional setup
+
+Before installing this extension, you'll need to:
+
+- [Set up Cloud Firestore in your Firebase project.](https://firebase.google.com/docs/firestore/quickstart)
+- Run Meilisearch instance. There are many easy ways [to download and run a Meilisearch instance.](https://docs.meilisearch.com/learn/getting_started/installation.html#download-and-launch)
```

Comment:
This is the 3th time you paste all these sections. Is there maybe a better way to avoid triple text repetition? If not no problem